src/app/globals.css

```
@@ -47,81 +48,319 @@
+  /* inherits :root Obsidian Night — no overrides needed */
 }
 
 @layer base {
   * {
-    @apply border-border outline-ring/50;
-  }
   body {
-    @apply text-foreground bg-[#050505] font-sans;
-    /* Micro-noise texture overlay */
-    background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 200 200' xmlns='http://www.w3.org/2000/svg'%3E%3Cfilter id='noiseFilter'%3E%3CfeTurbulence type='fractalNoise' baseFrequency='0.65' numOctaves='3' stitchTiles='stitch'/%3E%3C/filter%3E%3Crect width='100%25' height='100%25' filter='url(%23noiseFilter)' opacity='0.03'/%3E%3C/svg%3E");
+    `@apply` font-sans text-foreground;
```

Stylelint (17.13.0)

[error] Unknown word ` (CssSyntaxError)

(CssSyntaxError)